Formula/kibana.rb

```diff
@@ -15,8 +15,9 @@ class Kibana < Formula
   end
 
   resource "node" do
-    url "https://nodejs.org/dist/v7.2.0/node-v7.2.0.tar.xz" # N.B. includes vendored dependencies
-    sha256 "486d4db7ef659521ad2fafefca877638da07bef61e2aee090207ff52149294fb"
+    url "https://nodejs.org/dist/v7.2.1/node-v7.2.1.tar.xz" # N.B. includes vendored dependencies
+    mirror "https://npm.taobao.org/mirrors/node/latest-v7.x/node-v7.2.1.tar.xz"
```

Comment:
We probably don't, but https://bot.brew.sh/job/Homebrew%20Core/12921/version=any_sierra/console and on 10.11 and 10.10 too. Most likely a one time fluke but I didn't want to end up with a CI dud again the second time through.